cloudformation/elasticsearch.yaml

```diff
@@ -15,16 +15,17 @@ Resources:
         Ref: AWS::StackName
       UserPoolName: !Sub '${AWS::StackName}-Kibana'
       Schema:
```

Comment:
Now matching to be what it used to be: https://github.com/awslabs/fhir-works-on-aws-deployment/blob/mainline/cloudformation/elasticsearch.yaml#L14-L19